experimental/oauthtokenretriever/token.go

```diff
@@ -9,12 +9,12 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
```

Comment:
I guess `goimports` will be complaining here

Reply:
it doesn't complain so it should be valid :)